docs/lib/analytics/fragments/native_common/getting-started/common.md

```diff
@@ -0,0 +1,47 @@
+The Analytics category enables you to collect analytics data for your App. The Analytics category comes with built-in support for [Amazon Pinpoint](https://aws.amazon.com/pinpoint) and [Amazon Kinesis](https://aws.amazon.com/kinesis) (Kinesis support is currently only available in the Amplify JavaScript library).
+
+## Prerequisites
```

Comment:
missing goal here? maybe a simple record events section

Reply:
I don't know. For storage or API, there are a lot of functions call and sections. so can have a goal in getting-started section. But in Analytics, Record Event section if right follow getting-started, is it a bit repetitive?

Reply:
## Goal

To setup and configure your application with Amplify Analytics and record an event and user details.

Reply:
Pushed. Please take a look